airbyte-integrations/connectors/destination-dev-null/src/test-integration/kotlin/io/airbyte/integrations/destination/dev_null/CloudDevNullDestinationAcceptanceTest.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+package io.airbyte.integrations.destination.dev_null
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.airbyte.cdk.util.Jsons
+import io.airbyte.protocol.models.v0.AirbyteMessage
+import io.airbyte.protocol.models.v0.AirbyteRecordMessage
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+
+class CloudDevNullDestinationAcceptanceTest : TestingSilentDestinationAcceptanceTest() {
+    override val isCloudTest = true
+
+    override fun getConfig(): JsonNode {
```

Comment:
this is tweaked from the old DATs to use our new Jsons class. otherwise the files are completely unchanged from just `git checkout 7cdd998aa5~ -- <...path>`